impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js

```diff
@@ -60,8 +60,6 @@ define([
             plotFrameVisible: false,
             plotSizeMin: {width: 600, height: 300},
```

Comment:
Those are minimum sizes... So I'm ok with either approach (having global defaults or not).